BaseStation/Utilities/Utility.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuskyRobotics.Utilities
+{
+    public class Utility
```

Comment:
In my opinion this either needs to go back into the Networking code or needs to be moved to Scarlet. This method definitely needs documentation on what it is for. It is not immediately clear why this is being done (because we want to apply Math.Abs() directly afterwards).

Reply:
Good point.